### ISSUE TITLE
Cunning of the Cruel simming incorrectly for Elemental Shaman

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#1112]
Originally reported on Google Code with ID 1113

```
Note: this is not related to issue 1046.

While checking up on trinket rankings, I started to check the math behind the CotC
proc for Elemental Shaman. The sim results and my calculated values did not match,
nor do comparisons between sims and WoL parses for the same character.

Rough workings: https://docs.google.com/spreadsheet/ccc?key=0AvjwZoWzKatOdGRnODFhd3lLNHRTQmlfMjQxN0t2Rnc&hl=en_US#gid=1

In short, CotC is simming ~114% of what is observed via WoL, but even allowing for
some minor variation something is off.

Can you confirm how the damage values for CotC are calculated, and that the spell specific
& fire/frost/nature bonuses & talents for Elemental are not being used when calculating
CotC damage?
```

Reported by `binkenstein` on 2012-01-22 23:04:45
